R/app_ui.R

```diff
@@ -88,14 +88,23 @@ app_ui <- function() {
                                               ),
                              bs4Dash::tabItem(tabName = "data",
                                               shiny::fluidRow(
+                                                bs4Dash::column(12,
+                                                                shiny::numericInput("wordNumber",
+                                                                                   label = "Number of words:",
+                                                                                   value = 5,
+                                                                                   min = 0,
+                                                                                   max = 20                                                                )),
```

Comment:
It is still possible to add more than 20 words. I don't see a way to limit that from the `numericInput`. If there's no other way that this can be done quickly, I suggest we switch to a slider input ranging from 5 to 20.